install-guides/ubuntu22.md

```diff
@@ -0,0 +1,562 @@
+# Deploying Pixelfed on Ubuntu 22.04 LTS (Jammy Jellyfish) + PHP 8.1
```

Comment:
i'm not sure what the differences are between the 20.04 and 22.04 guide but either

a) make changes to both guides as you resolve the comments above, or
b) remove the 22.04 guide from the PR for now, fix the 20.04 guide, then copy it over to the 22.04 guide